fuzzers/074-dump_all/run_fuzzer.py

```diff
@@ -0,0 +1,127 @@
+import os
+import shutil
+import sys
+import subprocess
+import signal
+from multiprocessing import Pool
+from itertools import chain
+
+# Can be used to redirect vivado tons of output
+# stdout=DEVNULL in subprocess.check_call
+#try:
+#    from subprocess import DEVNULL
+#except ImportError:
+#    import os
+#    DEVNULL = open(os.devnull, 'wb')
+
+
+# Worker function called from threads
+def start_tiles(argList):
+    blockID, start, stop = argList
+    print("Running instance :" + str(blockID))
+    subprocess.check_call(
+        "${XRAY_VIVADO} -mode batch -source $FUZDIR/jobtiles.tcl -tclargs " +
+        str(blockID) + " " + str(start) + " " + str(stop),
+        shell=True)
+
+def start_nodes(argList):
+    blockID, start, stop = argList
+    print("Running instance :" + str(blockID))
+    subprocess.check_call(
+        "${XRAY_VIVADO} -mode batch -source $FUZDIR/jobnodes.tcl -tclargs " +
+        str(blockID) + " " + str(start) + " " + str(stop),
+        shell=True)
+
+# Function called once to get the total numbers of tiles to list
+def get_nb_tiles():
+    print("Fetching total number of tiles")
+    subprocess.check_call(
+        "${XRAY_VIVADO} -mode batch -source $FUZDIR/get_tilescount.tcl",
+        shell=True)
+    countfile = open("nb_tiles.txt", "r")
+    return int(countfile.readline())
+
+def get_nb_nodes():
+    print("Fetching total number of nodes")
+    subprocess.check_call(
+        "${XRAY_VIVADO} -mode batch -source $FUZDIR/get_nodescount.tcl",
+        shell=True)
+    countfile = open("nb_nodes.txt", "r")
+    return int(countfile.readline())
+
+
+def run_pool(itemcount, nbBlocks, blocksize, nbParBlock, workFunc):
```

Comment:
Should we move this function to prjxray/util.py and share it between 072 and 074?

Reply:
Yep it could be !

Reply:
I suggest we skip that for now and wait to make a refactoring work once it works for sure.